src/Cropper.tsx

```diff
@@ -103,29 +103,37 @@ class Cropper extends React.Component<CropperProps, State> {
   rafDragTimeout: number | null = null
   rafPinchTimeout: number | null = null
   wheelTimer: number | null = null
+  currentDoc: Document = document
+  currentWindow: Window = window
 
   state: State = {
     cropSize: null,
     hasWheelJustStarted: false,
   }
 
   componentDidMount() {
-    window.addEventListener('resize', this.computeSizes)
     if (this.containerRef) {
+      if (this.containerRef.ownerDocument) {
```

Comment:
I did also try adding some extra props to allow passing in of a window and document ref, but it seemed less efficient if using the likes of a `useRef` in the parent component to get this, as it took a couple of renders to get the `ref.current` set and then pass these down, so setting them in this component, pushing styles into header, etc. then needed to be handled in `componentDidUpdate`.

Reply:
If this approach fixed all the issue on Gutenberg side, it's great as it doesn't grow the component external API.

Just by curiosity, why trying to pass window and document refs instead of directly passing the window and document instances?

Reply:
> why trying to pass window and document refs instead of directly passing the window and document instances

we sort of have the same problem on the Gutenberg end in that within the component where `react-easy-crop` is used the default window and document references are the globals, so to get the iframe ones we have to work up from the current component with the same sort of `componenrRef.ownerDocument` approach - does that make sense?

Reply:
Yes I get it 👍

Reply:
If you are happy with this approach then I will get it tidied up and submitted for review tomorrow

Reply:
Alright, looking forward to it. 🎉